### DISCUSSION TITLE
The provided connection string is invalid or the server is inaccessible

### DISCUSSION THREAD

**Discussion by hehetiger** [OrchardCMS/OrchardCore#16615] [Q&A]
After downloading the code, set OrcardCore.Cms.Web as the startup option, press Ctrl+F5 to run it, and the browser will open the installation interface https://localhost:44300/ After entering all the text as required and clicking on 'Finish Installation', it will prompt that the provided connection string is invalid or the server is inaccessible

**Comment by hehetiger** [comment]
![企业微信截图_17246590118702](https://github.com/user-attachments/assets/20b20f4f-0596-4940-9fea-91f6c2ba629f)
Is there any incorrect information filled in

**Reply by duxonline** [reply]
I used to have to add `TrustServerCertificate=True` when installing on local:
`Server=server; Database=db; User Id=sa; Password=password; TrustServerCertificate=True`

**Reply by hehetiger** [reply]
I tried your suggestion, but it still didn't work.
I also tried 'Data Source=.;Initial Catalog=Orchard;Persist Security Info=True;User ID=sa;Password=123456;TrustServerCertificate=true' ,it didn't work either.
Do I need to execute certain commands to create a database first?

**Reply by duxonline** [reply]
Yeah, I remeber I created an empty database first.

**Reply by hehetiger** [reply]
What command to execute to create? Or just need to create an empty database whithout any tables?
